docs/source/serving/openai_compatible_server.md

```diff
@@ -111,6 +111,29 @@ directory [here](https://github.com/vllm-project/vllm/tree/main/examples/)
 :prog: vllm serve
 ```
 
+### Config file
+
+You may also supply these CLI args using a config file. For example:
+
+```yaml
+# config.yaml
+
+host: "127.0.0.1"
+port: 6379
+uvicorn-log-level: "info"
+```
+
+```bash
+$ vllm serve --model SOME_MODEL --config config.yaml
```

Comment:
`--model SOME_MODEL` can be in the `config.yaml` as well?

Reply:
done. edited the doc.